NitroxServer-Subnautica/GameLogic/Entities/Spawning/ReefbackBootstrapper.cs

```diff
@@ -38,7 +40,10 @@ public void Prepare(Entity parentEntity, DeterministicBatchGenerator determinist
                         for(int i = 0; i < totalToSpawn; i++)
                         {
                             NitroxId id = deterministicBatchGenerator.NextId();
-                            Entity child = new Entity(localSpawnPosition, new NitroxQuaternion(0, 0, 0, 1), new NitroxVector3(1, 1, 1), creature.techType.ToDto(), parentEntity.Level, creature.classId, true, id, null, parentEntity);
+                            NitroxTechType techType = aprilFools ? parentEntity.TechType : creature.techType.ToDto();
```

Comment:
Won't this result in infinite recursion?

Reply:
Nope, since the Reefback bootstrapper handles its own spawning it doesnt use the regular spawning code for spawning its children, thus no infinite recursion